pwa/app/routes/contact.tsx

```diff
@@ -0,0 +1,104 @@
+export default function Thanks(): JSX.Element {
+  return (
+    <>
+      <div className="flex flex-col divide-y [&_*]:mt-4">
+        <div>
+          <h1 className="text-3xl font-medium">Contact us</h1>
+          <p>
+            We&apos;d love to hear from you. Here&apos;s how to get in touch
+            about different issues or questions.
+          </p>
+        </div>
+
+        <div>
+          <h3 className="text-2xl">Is your team&apos;s information wrong?</h3>
+          <h4 className="text-xl">Team information</h4>
+          <p>
+            Most team data - like names, websites, sponsors, etc - are updated
+            directly from FIRST. If your team&apos;s information is incorrect on
+            The Blue Alliance, please check that it is correct on FIRST&apos;s
+            website. We update our records from{' '}
+            <a href="https://my.firstinspires.org/Dashboard/">
+              <em>FIRST</em>&apos;s Team Information Management System
+            </a>{' '}
+            daily, and are unable to change your team information to anything
+            other than what is listed there. You can ask your team&apos;s Lead
+            Mentor to update the information in FIRST TIMS and it will update on
+            The Blue Alliance within a day or two.
+          </p>
+          <h4 className="text-xl">Photos, Social Media, and Robot Names</h4>
+          <p>
+            Some team data - like photos, CAD models, social media, and robot
+            names - are provided to The Blue Alliance through crowdsourcing or
+            by teams directly. Teams can use the Team Moderator codes in the
+            FIRST Digital Kit of Parts to change robot names and approve or
+            change media on your own team page. You can get the code for your
+            team from the FIRST Digital Kit of Parts, and redeem it at the{' '}
+            <a href="https://thebluealliance.com/mod">
```

Comment:
The TBA links should be relative links (e.g. `/mod`). Also, use the `Link` component instead of `a`.

```
<Link to="/mod">TBA Team Mod dashboard</Link>
```